src/mainboard/clevo/tgl-u/bootblock.c

```diff
@@ -11,6 +11,7 @@
 #define DGPU_RST_N GPP_U4
 #define DGPU_PWR_EN GPP_U5
 
+#ifdef DRIVER_NVIDIA_OPTIMUS
 static void dgpu_power_enable(int onoff) {
```

Comment:
This function could be exposed by a header in the nvidia driver. Then I wouldn't need to be guarded by preprocessor

Reply:
Removed the `#ifdef` since it's not actually needed

Reply:
I meant define `void dgpu_power_enable(int onoff) ` in a header and let the mainboard implement it. without static you may implement it but not necessarily use